Makefile

```diff
@@ -83,6 +83,10 @@ modules: ## Runs go mod to ensure modules are up to date.
 	go mod tidy
 	cd $(TOOLS_DIR); go mod tidy
 
+.PHONY: generate
+generate: # Runs controller-gen for internal types for config file
+	controller-gen object paths="./pkg/config/v1alpha1/..."
```

Comment:
need to install controller-gen like we do golanci-lint (in the tools module)

Reply:
Good point!